daskhub/values.yaml

```
@@ -53,7 +53,7 @@ jupyterhub:
   singleuser:
     image:
       name: pangeo/base-notebook  # Image to use for singleuser environment. Must include dask-gateway.
-      tag: "2021.06.05"
+      tag: "2022.4.20"
```

```suggestion
      tag: "2022.04.20"
```

This was incorrectly adjusted from 2022.04.20 to 2022.4.20. 2022.04.20 is an invalid semver version, but is the actual tag. Maybe the action getting the last image adjusted this version tag somehow.